crates/sui-graphql-rpc/src/types/transaction_block_effects.rs

```diff
@@ -48,6 +48,7 @@ pub(crate) type CUnchangedSharedObject = Cursor<usize>;
 pub(crate) type CObjectChange = Cursor<usize>;
 pub(crate) type CBalanceChange = Cursor<usize>;
 
+/// The effects that are stored on-chain corresponding to a transaction block execution.
```

Comment:
Same here, I feel like TransactionBlockEffects is mostly self-explanatory, although mentioning that the information is stored on-chain might be useful since not everyone would know?

Reply:
"Effects that are stored on-chain" is not accurate -- these effects could correspond to a dry-run transaction, which is not stored on-chain.  These are simply the results of executing a transaction block.

Reply:
> these effects could correspond to a dry-run transaction

Good point, thank you!